### PR TITLE
Update version-check by a copy from PT page

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/version-check.rst [hunk 1]
```diff
@@ -4,63 +4,113 @@
...
-logged by the server. Stored information includes software version numbers and
-the unique ID of a checked system. The ID (an MD5 hash value) is generated
-either at installation or when the |version-check| query is done for the first
-time.
+As a result, the content of the sent query is as follows::
+
+  85624f3fb5d2af8816178ea1493ed41a;DBD::mysql;4.044
+  c2b6d625ef3409164cbf8af4985c48d3;MySQL;MySQL Community Server (GPL) 5.7.22-log
+  85624f3fb5d2af8816178ea1493ed41a;OS;Manjaro Linux
+  85624f3fb5d2af8816178ea1493ed41a;Percona::Toolkit;3.0.11-dev
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/version-check.rst:60]: Does PXB also sending PT version?

File: storage/innobase/xtrabackup/doc/source/version-check.rst [hunk 2]
```diff
@@ -4,63 +4,113 @@
...
-you might prefer to disable this feature, either onetime or permanently.
-To disable it onetime, use ``--no-version-check`` option when invoking the tool
-from a Percona product which supports it.
+you might prefer to disable this feature, either one time or permanently.
+To disable it one time, use ``--no-version-check`` option when invoking the
+tool from a Percona product which supports it. Here is a simple example which
+shows running `pt-diskstats <https://www.percona.com/doc/percona-toolkit/LATEST/pt-diskstats.html>`_
+tool from the |pt| with |version-check| turned off::
+
+  pt-diskstats --no-version-check
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/version-check.rst:73]: Need to replace with example running xtrabackup?

File: storage/innobase/xtrabackup/doc/source/version-check.rst [hunk 3]
```diff
@@ -4,63 +4,113 @@
...
+
+Each version check request is logged by the server. Stored information consists
+of the checked system unique ID followed by the software name and version.
+The ID is generated either at installation or when the |version-check| query is
+submitted for the first time.
+
+.. note::
 
-Then, it checks for and warns about versions with known problems.
+   Prior to version 3.0.7 of |pt|, the system ID was calculated as an MD5 hash
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/version-check.rst:51]: Version check script in xtrabackup is outdated and still using MD5(hostname)

File: storage/innobase/xtrabackup/doc/source/version-check.rst [hunk 4]
```diff
@@ -4,63 +4,113 @@
...
 
 Usage
------
+=====
 
 *Version Check* was implemented in |pt| 2.1.4, and was enabled by default in
-version 2.2.1. Currently it is supported by most tools in |pt|, |pxb|, and
-|pmm|. In |Percona XtraBackup|, this feature was introduced in version 2.1.5. By
-default, it is also enabled.
+version 2.2.1. Currently it is supported as a ``--[no]version-check`` option
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/version-check.rst:25]: Speaking of xtrabackup, version check has been added in 2.1.5 and was always enabled by default.

**Review by gl-sergei — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/version-check.rst (hunk 1 above)
Comment (dbadapt) [reply storage/innobase/xtrabackup/doc/source/version-check.rst:60]: Sergei, yes, the version check script in the PXB repo reports toolkit version.   Sadly,  it is not configured to report PXB version.   There is a another jira ticket for this: PXB-1615

Hopefully we will get this resolved before the PXB 8.0.1 release

File: storage/innobase/xtrabackup/doc/source/version-check.rst (hunk 3 above)
Comment (dbadapt) [reply storage/innobase/xtrabackup/doc/source/version-check.rst:51]: yes, this page is a copy of the same version check page from toolkit.

File: storage/innobase/xtrabackup/doc/source/version-check.rst (hunk 4 above)
Comment (dbadapt) [reply storage/innobase/xtrabackup/doc/source/version-check.rst:25]: I think multi-purpose toolkit/xtrabackup version of this document is ok for now.

**Comment by dbadapt** [comment]
LGTM

**Pull request merged**